client/src/AppBundle/Resources/translations/report-assets.en.yml

```diff
@@ -17,7 +17,7 @@ existPage:
       label: Save and continue
 
 typePage:
-  htmlTitle: "Deputy report - assets | GOV.UK"
```

Comment:
Similar to previous comment. May need a different name so as not to confuse users.

Reply:
Believe this relates to the other comment I made around the title of the page 👍🏻